src/test/java/com/spotify/docker/client/DefaultDockerClientUnitTest.java

```diff
@@ -209,6 +209,16 @@ public void testHostWithNonProxyHost() {
       assertThat((String) client.getClient().getConfiguration()
                       .getProperty("jersey.config.client.proxy.uri"),
               isEmptyOrNullString());
+      final DefaultDockerClient client1 = DefaultDockerClient.builder()
```

Comment:
Just curious what these additional lines are testing?

Is it easy to add a test for the quote-stripping logic above?

Reply:
Hello David Xia,
Before making changes in docker-client I encountered the problem that HTTP requests to localhost / 127.0.0.1 were sent via proxy. It was quite unexpected for me to learn so.
So, these test ensures that requests to 127.0.0.1 and localhost never go through proxy server..